sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -136,28 +184,35 @@
 <SALTelemetry>
     <Subsystem>MTM2</Subsystem>
     <EFDB_Topic>MTM2_temperaturesMeasured</EFDB_Topic>
```

Comment:
Consider renaming this to `temperature` only...

Reply:
This is based on the LTS-162 v5.